src/hooks/hover.js

```diff
@@ -0,0 +1,24 @@
+// source: https://usehooks.com/useHover/
```

Comment:
@brandonmcconnell this could be that you are much more senior than me and there is a good reason for all of this, but are we sure this is really necessary for implementing the hover effect? Feel like there should be a simpler way to do it with the styles, but perhaps not actually. I will defer to you on this, but just wanted to comment and check.

Reply:
Yes, absolutely. Using plain CSS< this will be much simpler, so I'll remove this hook when I move the styles over to pure CSS 👍🏼